docs/styles/global.css

```diff
@@ -123,7 +123,7 @@ sl-menu::part(base) {
 }
 
 sl-menu-item::part(base):hover {
```

Comment:
We also probably need the ring that shows on hover to be more visible. I achieved this on darker buttons by putting the following:
border: 1px solid white;
outline: 2px solid black;

This way you kind of get a gap and then the ring, so that it is more visible (See the package for stores button on preview to see it in action)